source/assets/sass/components/_link-header.scss

```diff
@@ -76,7 +76,5 @@ h6 {
 
   to {
     background-color: #f8f8f8;
```

Comment:
I pulled this directly from the (now-removed) Rouge theme, but feel free to replace this with a CSS var or whatever.